<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UusiTiedot</class>
 <widget class="QWidget" name="UusiTiedot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nimi</string>
     </property>
     <property name="buddy">
      <cstring>nimiEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nimiEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Katuosoite</string>
     </property>
     <property name="buddy">
      <cstring>osoiteEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="osoiteEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Pankkitili (IBAN)</string>
     </property>
     <property name="buddy">
      <cstring>tiliLine</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="tiliLine">
     <property name="placeholderText">
      <string>FI0000000000000000</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="muotoLabel">
     <property name="text">
      <string>Yritysmuoto</string>
     </property>
     <property name="buddy">
      <cstring>muotoList</cstring>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QListWidget" name="muotoList"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Tilikartan laajuus</string>
     </property>
     <property name="buddy">
      <cstring>laajuusList</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QListWidget" name="laajuusList"/>
   </item>
   <item row="15" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>80</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y-tunnus</string>
     </property>
     <property name="buddy">
      <cstring>ytunnusEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ytunnusEdit">
     <property name="maxLength">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Kotipaikka</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="kotipaikkaEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Postiosoite</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
     <item>
      <widget class="QLineEdit" name="postinumeroEdit">
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="kaupunkiEdit"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Sähköpostiosoite</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Kotisivu</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="emailEdit"/>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="webEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Puhelinnumero</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="puhelinEdit"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="bicLabel">
     <property name="text">
      <string>BIC</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="pankkiLabel">
     <property name="text">
      <string>Pankki</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="bicEdit"/>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="pankkiEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nimiEdit</tabstop>
  <tabstop>ytunnusEdit</tabstop>
  <tabstop>osoiteEdit</tabstop>
  <tabstop>postinumeroEdit</tabstop>
  <tabstop>kaupunkiEdit</tabstop>
  <tabstop>kotipaikkaEdit</tabstop>
  <tabstop>puhelinEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>webEdit</tabstop>
  <tabstop>tiliLine</tabstop>
  <tabstop>muotoList</tabstop>
  <tabstop>laajuusList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
